cpp/oneapi/dal/algo/decision_forest/backend/gpu/train_kernel_reg_hist_dpc.cpp

```diff
@@ -32,15 +32,20 @@ template <typename Float>
 static result_t call_daal_kernel(const context_gpu& ctx,
                                  const descriptor_t& desc,
                                  const table& data,
-                                 const table& responses) {
+                                 const table& responses,
+                                 const table& weights) {
```

Comment:
If I remember correctly this now matches CPU side for const table& weights even in an unweighted case right?

Reply:
Yes, that's true